src/definitions.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    info: {
+   
+      title: 'The Alcoding Club Application Api\'s',
+      version: '1.0.0',
+      description: 'All Backend Api\'s end points', 
+    },
+    host:'localhost:3000', // Host 
```

Comment:
Can we pick up this information from config.js. Do have a talk with @temanisparsh.

Reply:
okay will update you once done